### PR TITLE
Hide dev enviroments

### PR DESCRIPTION
**Pull request by AaronCrawfis**
- Hides the inner loop section
- Removes dev/Azure environments from Dapr quickstart and tutorial
- Cleans up FAQ page

### CODE REVIEW

File: docs/content/reference/faq.md [hunk 1]
```diff
@@ -6,11 +6,6 @@ description: "Commonly asked questions about best practices"
 weight: 999
 ---
 
-<!-- TODO: this page needs an overhaul. transition it from our old FAQ page to Best practices. 
-also see - https://github.com/project-radius/docs/issues/93
--->
-
-
 ## Appllications
```
Comment (Reshrahim) [inline docs/content/reference/faq.md]: *nitpick* - typo

File: docs/content/reference/faq.md [hunk 2]
```diff
@@ -21,7 +16,7 @@ also see - https://github.com/project-radius/docs/issues/93
 
 ### Can I connect to an existing environment?
 
-**Yes**. When you initialize an environment via [`rad env init`]({{< ref rad_env_init.md >}})), you can provide an existing Azure subscription or Kubernetes cluster context. Radius will update your `config.yaml` file with the appropriate values.
+**Yes**. When you initialize an environment via [`rad env init`]({{< ref rad_env_init.md >}})), you can provide an existing Kubernetes cluster context. Radius will update your `config.yaml` file with the appropriate values.
```
Comment (Reshrahim) [inline docs/content/reference/faq.md:19]: Is the name still config.yaml with workspaces change? I belive the structure of the contents changed

Comment (AaronCrawfis) [reply docs/content/reference/faq.md:19]: I believe so, yes. Just the contents changed.

File: docs/content/reference/faq.md (hunk 1 above)
Comment (AaronCrawfis) [reply docs/content/reference/faq.md]:
```suggestion
## Applications
```

**Review by Reshrahim — APPROVED** [review]

**Pull request merged**